eng/SourceBuildPrebuiltBaseline.xml

```diff
@@ -0,0 +1,11 @@
+<!-- When altering this file or making other Source Build related changes, include @dotnet/source-build as a reviewer. -->
+<!-- See aka.ms/dotnet/prebuilts for guidance on what pre-builts are and how to eliminate them. -->
+
+<UsageData>
+  <IgnorePatterns>
+     <!-- These are coming in via runtime but the source-build infra isn't able to automatically pick up the right intermediate. -->
+    <UsagePattern IdentityGlob="Microsoft.NET.ILLink.Tasks/*9.0.*" />
```

Comment:
@MichaelSimons this is copied from asp.net core repo, without it I get warning about pre-built, please let me know if this is acceptable.